specification/cognitiveservices/data-plane/Luis/v2.0/LUIS Endpoint API.swagger.json

```diff
@@ -54,51 +54,58 @@
   "paths": {
     "/{appId}": {
       "get": {
-        "description": "Gets the published endpoint predictions for the given query",
-        "operationId": "v2.0 Response - Get predictions from endpoint",
+        "description": "Gets the published endpoint predictions for the given query.",
+        "operationId": "Luis_GetPredictionsFromEndpointViaGet",
         "parameters": [
           {
             "name": "appId",
             "in": "path",
-            "description": "Format - guid. The application ID",
+            "description": "Format - guid. The application ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "subscription-key",
```

Comment:
`"subscription-key"` parameter is a temporary addition used so autorest-generated SDK would be operational.